src/Bridge/Symfony/Bundle/Resources/views/DataCollector/request.html.twig

```diff
@@ -60,6 +67,12 @@
             <tbody>
                 {% for key, itemOperation in collector.resourceMetadata.itemOperations %}
                     {{ apiPlatform.operationLine(key, itemOperation, collector.requestAttributes.item_operation_name|default('')) }}
+                {% else %}
```

Comment:
maybe this block is not needed as there is always at least one item operation so the IRI can be generated, is this assumption correct ?

Reply:
Yes, you can drop it.